src/rules/function-url-quotes/__tests__/index.js

```diff
@@ -426,3 +428,14 @@ testRule(rule, {
     column: 21,
   } ],
 })
+
+testRule(rule, {
+  ruleName,
+  config: [ "always", { except: ["empty"] } ],
+
+  accept: [ {
+    code: "@-moz-document url-prefix() {}",
+  }, {
+    code: "a { background: url() }",
+  } ],
```

Comment:
Can we get one simple reject test here to show that *non empty* functions are still checked please e.g.

```js
reject: [{
    code: "@import url(foo.css);",
    message: messages.expected(),
    line: 1,
    column: 13,
  }]
```